### PR TITLE
feat:DBZ-3137 Add option to view the running connector configuration

### PR DESCRIPTION
**Pull request by indraraj**
Jira issue:- https://issues.redhat.com/browse/DBZ-3137

### CODE REVIEW

**Comment by indraraj** [comment]
This https://issues.redhat.com/browse/DBZ-4627 need to be implemented before merging it.

File: ui/packages/ui/src/locales/it/public.json [hunk 1]
```diff
@@ -13,6 +13,12 @@
   "addDefinition": "+ Aggiungi definizione",
   "addDefinitionTooltip": "Aggiungere un'altra riga definizione",
   "removeDefinitionTooltip": "Rimuovi questa riga di definizione",
+  "show hidden fields icon": "show hidden fields icon",
+  "Hide password": "Hide password",
+  "Show password": "Show password",
+  "Successfully copied to clipboard!": "Successfully copied to clipboard!",
+  "Copy to clipboard": "Copy to clipboard",
+  "Download JSON": "Download JSON",
```
Comment (mdrillin) [inline ui/packages/ui/src/locales/it/public.json:21]: Entries in this file need to be translated.

File: ui/packages/services/src/connector/connector.service.ts [hunk 2]
```diff
@@ -138,6 +138,16 @@ export class ConnectorService extends BaseService {
         return this.httpGet<Connector[]>(endpoint);
     }
 
+    /**
+     * Get the Connector config
+     */
+     public getConnectorConfig(clusterId: number, connectorName: string): Promise<any> {
+        this.logger?.info("[ConnectorService] Fetch the connector");
+
+        const endpoint: string = this.endpoint("/api/connectors/:clusterId/:connectorName/config", { clusterId, connectorName });
```
Comment (mdrillin) [inline ui/packages/services/src/connector/connector.service.ts:147]: The leading '/api' needs to be removed

**Review by mdrillin — COMMENTED** [review]
Overall this looks really good!  The PR will need to be rebased, and I also made a couple comments.  I will finish up and submit the backend service PR separately.

**Comment by indraraj** [comment]
can be merged along with https://github.com/debezium/debezium-ui/pull/512

**Comment by mdrillin** [comment]
retested after changes to #512 - merging

**Pull request merged**